repositories.yaml

```
@@ -1083,3 +1083,25 @@ applications:
       - v1_name: rally-attention-stream
         app_id: rally-attention-stream
     skip_documentation: true
+
+- app_name: viu_politica
+    canonical_app_name: Viu Politica
+    app_description: |
+      This is a "spin-off" of the Mozilla YouTube RegretsReporter extension. It is an opt-in browser extension where users can anonymously tag videos related to the 2022 Brazilian Elections.
```

It's a multi-line field, so let's keep column widths down.

```suggestion
      This is a "spin-off" of the Mozilla YouTube RegretsReporter extension.
      It is an opt-in browser extension where users can anonymously tag videos related to the 2022 Brazilian Elections.
```